app/scripts/background.js

```diff
@@ -753,13 +754,22 @@ export function setupController(
   }
 
   function getUnapprovedTransactionCount() {
-    const pendingApprovalCount =
-      controller.approvalController.getTotalApprovalCount();
+    console.log('getUnapprovedTransactionCount called');
     const waitingForUnlockCount =
       controller.appStateController.waitingForUnlock.length;
-    return pendingApprovalCount + waitingForUnlockCount;
+    const queuedRequestsCount = controller.queuedRequestController.length();
+
+    const pendingApprovalCount =
+          controller.approvalController.getTotalApprovalCount();
+
+    return waitingForUnlockCount + pendingApprovalCount + queuedRequestsCount;
```

Comment:
this is currently giving the wrong number (its always off by 1 because the pendingApprovalCount & queuedRequesCount end up double counting things =S)